ui/models/job.js

```diff
@@ -20,22 +20,20 @@ export default class JobModel {
     // endpoint e.g. the similar jobs endpoint. It defaults to the job
     // list endpoint.
     const { fetchAll, uri: configUri } = config;
-    const jobUri = configUri || getProjectUrl(uri);
+    const jobUri = configUri || getApiUrl(uri);
     const { data, failureStatus } = await getData(
       `${jobUri}${options ? createQueryParams(options) : ''}`,
     );
 
     if (!failureStatus) {
-      const { results, meta, job_property_names } = data;
+      const { results, job_property_names, next } = data;
       let itemList;
       let nextPagesJobs = [];
 
-      // if the number of elements returned equals the page size,
-      // fetch the next pages
-      if (fetchAll && results.length === meta.count) {
-        const count = parseInt(meta.count, 10);
-        const offset = parseInt(meta.offset, 10) + count;
-        const newOptions = { ...options, offset, count };
+      // if ``next`` gives a URL for more, fetch the next pages
+      if (fetchAll && next) {
```

Comment:
`nextUrl` would be clearer and then maybe eliminate the need for a comment?

Reply:
Good point.  Done...